### PR TITLE
chore: add spellchecker

### PR DESCRIPTION
**Pull request by Vritra4**
# Description

add a workflow to check spell

---

## Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title, you can find examples of the prefixes below:
  
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] included the necessary unit and integration tests
- [ ] updated the relevant documentation or specification, including comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] confirmed all CI checks have passed

## Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic, API design and naming, documentation is accurate, tests and test coverage

### CODE REVIEW

File: x/move/client/cli/utils_test.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ func Test_readJSONStringArray(t *testing.T) {
 	testCases := []struct {
 		name   string
 		s      string
-		expRes []string
+		express []string
```
Comment (beer-1) [inline x/move/client/cli/utils_test.go:16]: expectedResponse

File: x/bank/keeper/msg_server.go [hunk 2]
```diff
@@ -102,7 +102,7 @@ func (k msgServer) MultiSend(goCtx context.Context, msg *types.MsgMultiSend) (*t
 
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	// NOTE: totalIn == totalOut should already have been checked
+	// NOTE: total In == total Out should already have been checked
```
Comment (beer-1) [inline x/bank/keeper/msg_server.go:105]: hmm not valid fix, but good to keep anyway

File: x/move/keeper/balancer.go [hunk 3]
```diff
@@ -206,7 +206,7 @@ func (k BalancerKeeper) poolMetadata(ctx context.Context, metadataLP vmtypes.Acc
 		return nil, err
 	}
 
-	metadataA, _, err := k.Keeper.MoveBankKeeper().Balance(ctx, storeA)
+	metadata, _, err := k.Keeper.MoveBankKeeper().Balance(ctx, storeA)
```
Comment (beer-1) [inline x/move/keeper/balancer.go:209]: should keep metadataA to indicate it is metadata of coin A

File: x/bank/keeper/msg_server.go (hunk 2 above)
Comment (Vritra4) [reply x/bank/keeper/msg_server.go]: added totalIn and totalOut on ignorelist

File: x/move/client/cli/utils_test.go (hunk 1 above)
Comment (Vritra4) [reply x/move/client/cli/utils_test.go]: reverted

File: x/move/keeper/balancer.go (hunk 3 above)
Comment (Vritra4) [reply x/move/keeper/balancer.go]: reverted

File: x/move/client/cli/utils_test.go [hunk 4]
```diff
@@ -30,7 +30,7 @@
 			require.Error(t, err, tc.name)
 		} else {
 			require.NoError(t, err, tc.name)
-			require.Equal(t, tc.expRes, res, tc.name)
+			require.Equal(t, tc.express, res, tc.name)
```
[the remark this answers is not in the document]
Comment (Vritra4) [reply x/move/client/cli/utils_test.go]: reverted

File: x/move/client/cli/utils_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (Vritra4) [reply x/move/client/cli/utils_test.go]: reverted

File: x/move/client/cli/utils_test.go [hunk 5]
```diff
@@ -30,17 +30,17 @@ func Test_readJSONStringArray(t *testing.T) {
...
 	}
 }
 
 func Test_decodeJSONStringArray(t *testing.T) {
 	testCases := []struct {
-		name   string
-		ss     []string
+		name    string
+		ss      []string
 		express []string
```
Comment (beer-1) [inline x/move/client/cli/utils_test.go:42]: This too expRes

**Review by beer-1 — APPROVED** [review]
LGTM except last comment 👍

File: x/move/client/cli/utils_test.go (hunk 5 above)
Comment (Vritra4) [reply x/move/client/cli/utils_test.go]: thanks! also reverted

**Pull request merged**